deploy/deployment.yaml

```diff
@@ -0,0 +1,24 @@
+---
+kind: Deployment
+apiVersion: extensions/v1beta1
+metadata:
+  name: freenas-nfs-provisioner
+  namespace: kube-system
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: freenas-nfs-provisioner
+    spec:
+      serviceAccountName: freenas-nfs-provisioner
+      containers:
+        - name: freenas-nfs-provisioner
+          image: docker.io/nmaupu/freenas-provisioner:2.0
```

Comment:
@nmaupu make note of tag here.  Let me know if you want something else.

Reply:
That's what I will use for the release tag.